[UPD] Switch to Github Actions

rousseldenis opened the pull request.

rousseldenis committed:
[UPD] Copier Update

rousseldenis committed:
[UPD] Switch to Github Actions

pedrobaeza commented:
This one is not going to be easy...

rousseldenis committed:
[IMP] Don't fail on license check

rousseldenis commented:
> This one is not going to be easy...

Not so hard.

Deactivated license mandatory check

rousseldenis commented:
@LoisRForgeFlow @JordiBForgeFlow purchase_order_approval_block is LGPL but based on purchase_exception which is AGPL. This is forbiddden. IMHO you should go to AGPL

rousseldenis commented:
@pedrobaeza :green_apple:

pedrobaeza commented:
Great, I think we should directly switch that modules to AGPL and don't deactivate license check, but you mark the rhythm.

rousseldenis commented:
> Great, I think we should directly switch that modules to AGPL and don't deactivate license check, but you mark the rhythm.

I usually ping the authors to warn them as I'd prefer to have their feeling about that.

Even if you are right, this check should pass anyway

rousseldenis commented:
Following this https://github.com/odoo/odoo/commit/49b143e1b7f12ee6cf4d14754904f3e9b64a1011

It seems we don't need anymore purchase_order_line_sequence module.

@damdam-s @BernatPForgeFlow

rousseldenis committed:
[REM] purchase_order_line_sequence: Not needed anymore

After https://github.com/odoo/odoo/commit/49b143e1b7f12ee6cf4d14754904f3e9b64a1011

the sequence is correctly transmitted from purchase line to invoice line.

rousseldenis commented:
@pedrobaeza What do we do here ? Let license and ignore or change license ?

pedrobaeza commented:
I would switch the license, as the author are warned and can make any later move, but keep the check for possible new contributions.

rousseldenis committed:
[CHG] purchase_order_approval_block: Change to AGPL as depending on AGPL purchase_exception

rousseldenis commented:
> I would switch the license, as the author are warned and can make any later move, but keep the check for possible new contributions.

Done

rousseldenis committed:
[CHG] purchase_minimum_amount: Change to AGPL as depending on AGPL purchase_order_approval_block

pedrobaeza commented:
/ocabot merge patch

matteoopenf commented on purchase_order_line_sequence/__manifest__.py:
```diff
@@ -24,7 +24,7 @@
     ],
     "post_init_hook": "post_init_hook",
-    "installable": True,
+    "installable": False,
```
why this key became false?
Which is the problem?

matteoopenf replied:
https://github.com/OCA/purchase-workflow/pull/1564
